src/plugins/support/PMRSupport/src/pmrsupportplugin.cpp

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+
+Licensed to the OpenCOR team under one or more contributor license agreements.
+See the NOTICE.txt file distributed with this work for additional information
+regarding copyright ownership. The OpenCOR team licenses this file to you under
+the Apache License, Version 2.0 (the "License"); you may not use this file
+except in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+*******************************************************************************/
+
+//==============================================================================
+// PMR Support plugin
+//==============================================================================
+
+#include "coreguiutils.h"
+#include "pmrsupportplugin.h"
+
+//==============================================================================
+
+#include <Qt>
+
+//==============================================================================
+
+#include <QMainWindow>
+#include <QSettings>
+
+//==============================================================================
+
+namespace OpenCOR {
+namespace PMRSupport {
+
+//==============================================================================
+
+PLUGININFO_FUNC PMRSupportPluginInfo()
+{
+    Descriptions descriptions;
+
+    descriptions.insert("en", QString::fromUtf8("a plugin to access the <a href=\"https://models.physiomeproject.org/\">Physiome Model Repository</a>."));
+    descriptions.insert("fr", QString::fromUtf8("une extension pour accéder au <a href=\"https://models.physiomeproject.org/\">Répertoire de Modèles Physiome</a>."));
+
+    return new PluginInfo("Support", true, false,
+                          QStringList() << "Core" << "libgit2" << "zlib",
+                          descriptions,
+                          QStringList());
+}
+
+//==============================================================================
+// Plugin interface
```

Comment:
You don't handle any of the plugin interfaces, so no need to support that interface. In other words, it looks like ```PMRSupport``` doesn't need to support any interface at all (i.e. like ```ZIPSupport```).